cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflow.java

```diff
@@ -106,4 +106,12 @@ protected final void addMapReduce(String mapReduce) {
   protected final void addSpark(String spark) {
     configurer.addSpark(spark);
   }
+
+  /**
+   * Adds a fork to the {@link Workflow}
+   * @return the {@link WorkflowForkConfigurer}
```

Comment:
Should be a bit more descriptive. Something like `a {@link WorkflowForkConfigurer} to configure actions in the fork`